cassandra-mesos-scheduler/src/main/java/io/mesosphere/mesos/frameworks/cassandra/scheduler/CassandraCluster.java

```diff
@@ -93,7 +93,8 @@
     );
 
     private static final Map<String, String> executorEnv = unmodifiableHashMap(
-        tuple2("JAVA_OPTS", "-Xms256m -Xmx256m")
+        tuple2("JAVA_OPTS", "-Xms256m -Xmx256m"),
+        tuple2("MESOSPHERE_LD_LIBRARY_PATH", "/opt/mesosphere/lib")
```

Comment:
On non dcos clusters (most major linux distributions), LD library path is typically located under "/usr/local/lib". I'm not very familiar with this code, but I wanted to double check that the default value is correct here.